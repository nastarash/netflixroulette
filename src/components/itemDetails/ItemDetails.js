import React from 'react';

class ItemDetails extends React.Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return (
      <div>
        <img src="" alt="bla-bla" />
        <h3>Tile</h3>
        <span>date</span>
        <span>genre</span>
      </div>
    );
  }
}

export default ItemDetails;
