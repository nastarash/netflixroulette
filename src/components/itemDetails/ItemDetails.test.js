import React from 'react';
import renderer from 'react-test-renderer';
import ItemDetails from './ItemDetails';

it('render correctly', () => {
  const AppComponent = renderer.create(<ItemDetails />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
