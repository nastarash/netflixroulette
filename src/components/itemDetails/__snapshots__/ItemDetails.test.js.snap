// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div>
  <img
    alt="bla-bla"
    src=""
  />
  <h3>
    Tile
  </h3>
  <span>
    date
  </span>
  <span>
    genre
  </span>
</div>
`;
